components/Terms/Edit.js

```diff
@@ -12,21 +12,31 @@ export default class Edit extends Component {
 	static propTypes = {
 		term: PropTypes.Term.isRequired,
 	}
+
+	constructor(props) {
+		super(props)
+		this.state = {
```

Comment:
@pdewouters why is this copying to the state? Let's just use the term directly in `this,props` in `render`